spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context 'methods' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.